ethereum/core/src/test/java/org/hyperledger/besu/ethereum/core/TransactionEIP1559Test.java

```diff
@@ -1,79 +0,0 @@
-/*
```

Comment:
Why should we remove this test?

Reply:
because it is not an unit test, there are not assert and it is like a _main_ to run some code